src/test/conditions/PreimageSha256_test.cpp

```diff
@@ -333,6 +333,7 @@ class PreimageSha256_test : public beast::unit_test::suite
                 log << "Fulfillment deserialize error: " << std::get<0>(x) << " " << ec.message() << '\n';
             }
             auto const c = Condition::deserialize (hexblob(std::get<2>(x)), ec);
```

Comment:
The condition here does not match the fulfillment. They were just dummy values used to test the parser (if you look at the fingerprint in the condition, it's  `4849505152535455484950515253545548`, which is clearly not a valid fingerprint.

The history is this test predated the generated tests (which _do_ check that the conditions match the fulfillments), and I decided to keep the old tests. Having said all that, I do agree this should be improved. Now that I have python scripts, I'll fix the conditions so they match the fulfillments as expected and change the test. A second option would be to just remove this file (as the generated tests are better).